Packs/DuoAdminApi/Integrations/DuoAdminApi/DuoAdminApi.py

```diff
@@ -8,7 +8,7 @@
 
 HOST = demisto.getParam('hostname')
 INTEGRATION_KEY = demisto.getParam('integration_key')
```

Comment:
I think the INTEGRATION_KEY should be part of the credentials (the identifier). Please verify and update accordingly

Reply:
updated